cvat/apps/quality_control/quality_reports.py

```diff
@@ -1971,17 +1973,23 @@ def _find_closest_unmatched_shape(shape: dm.Annotation):
     _UNMATCHED_IDX = -1
 
     def _make_zero_confusion_matrix(self) -> Tuple[List[str], np.ndarray]:
-        label_names = [
-            label.name
-            for label in self._gt_dataset.categories()[dm.AnnotationType.label]
+        label_id_name_map = {
```

Comment:
This map is never actually used as a map.

IMO, it would be clearer to rewrite this comprehension as an imperative loop and just build the name list and id-to-index map simultaneously:

```python
for label_id, label in enumerate(self._gt_dataset.categories()[dm.AnnotationType.label])
    if not label.parent:
         label_names.append(label.name)
         label_id_idx_map[label_id] = label_idx
         label_idx += 1
```